components/style/custom_properties.rs

```diff
@@ -613,32 +615,39 @@ fn substitute_all(
 fn substitute_one(
     name: &Name,
     specified_value: &BorrowedSpecifiedValue,
-    specified_values_map: &OrderedMap<&Name, BorrowedSpecifiedValue>,
+    specified_values_map: &OrderedMap<&Name, Option<BorrowedSpecifiedValue>>,
     partial_computed_value: Option<&mut ComputedValue>,
     custom_properties_map: &mut CustomPropertiesMap,
     invalid: &mut PrecomputedHashSet<Name>,
 ) -> Result<TokenSerializationType, ()> {
-    if let Some(computed_value) = custom_properties_map.get(&name) {
-        if let Some(partial_computed_value) = partial_computed_value {
-            partial_computed_value.push_variable(computed_value)
-        }
-        return Ok(computed_value.last_token_type)
-    }
```

Comment:
Why do we remove this block?

Reply:
This function is completely reworked in the next commit, but yeah, this was causing the tests failures in https://github.com/servo/servo/pull/18785. Anyway, all of it is gone now, given we now have stronger invariants about which code arrives here.